File: presentation-examples/function-definition.java
import sqlx.connection.PgConnection;
import reqwest.Client;

public class Client {
    private Client reqwest;
    private PgConnection conn;

    public String make_request(String url) {
        String response = this.reqwest
            .post(url)
            .send()
            .text;
        
        return text;
    }
}

File: presentation-examples/function-definition.rs
use sqlx::connection::PgConnection;

pub struct Client {
    reqwest: reqwest::Client,
    conn: PgConnection,
}

impl Client {
    pub fn make_request(&self, url: String) -> String {
        let response: String = self.reqwest
            .post(&url)
            .send()
            .text;
        
        return text;
    }
}

